LangChain/LangChainComponentImplementation.md

```
@@ -0,0 +1,112 @@
+### PromptTemplate
+
+## String Prompt Template:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)